### ISSUE TITLE
Use dbuild to ensure TypelevelDeps work together. 

### ISSUE THREAD

**Issue by InTheNow** [InTheNow/sbt-catalysts#11]
See https://github.com/typesafehub/dbuild

>... to coordinate the development of multiple, independent projects that evolve in parallel...and makes sure that they all work together, even though each of them may evolve independently.

**Comment by InTheNow** [comment]
Also see https://gitter.im/scala/slip?at=562360f903c6b52d3c598f07

**Comment by milessabin** [comment]
Lot's of directions we could take this. I would love to be able to specify a set of components (compiler, libraries, etc.) by git hashes and have the complete set built together as a consistent bundle.

**Comment by InTheNow** [comment]
Ok - I suggest as a first run that I use dbuild to build the libs in https://github.com/InTheNow/sbt-catalysts/blob/master/src/main/scala/org/typelevel/TypelevelDeps.scala, minus paroadise, scalac, specs2 etc and take it from there

**Comment by milessabin** [comment]
:+1:

**Comment by dwijnand** [comment]
A few months back there were necessary pieces used by dbuild that weren't publicly available, making it near impossible to use. But that could've an extreme view and/or no longer relevant. So I guess I'm saying it might not be possible, but hopefully it will.

**Comment by InTheNow** [comment]
@dwijnand Thanks for the insider heads-up - is there anyone that might have a more concrete view on this?

**Comment by dwijnand** [comment]
I think Seth's your best bet.

**Comment by InTheNow** [comment]
>  and have the complete set built together as a consistent bundle.

@milessabin Is this for testing only, or releasing as a Pom with dependencies, or one big Jar, or..?

**Comment by milessabin** [comment]
Ultimately? For releasing (or internal use) in some form or another. But let's walk before trying to run :-)
